src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -39,6 +39,7 @@
     <script type="text/javascript" src="<%= FrontEndLibrary.JQUERY_UI %>"></script>
     <script type="text/javascript" src="<%= FrontEndLibrary.BOOTSTRAP %>"></script>
     <script type="text/javascript" src="<%= FrontEndLibrary.BOOTBOX %>"></script>
+    <script type="text/javascript" src="/js/const.js"></script>
```

Comment:
Since we are doing this, we can very soon move them to ES6 (is it ready?) using const keyword